content/en/docs/reference/glossary/event.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Event
+id: event
+date: 2022-01-16
+full_link: 
+short_description: >
+   Event is a report of an event somewhere in the cluster. It generally denotes some state change in the system.
+aka: 
+tags:
+- core-object
+- fundamental
+---
+Event is a report of an event somewhere in the {{< glossary_tooltip text="cluster" term_id="cluster" >}}. 
+It generally denotes some state change in the system.
+
+<!--more-->
+Events have a limited retention time and triggers and messages may evolve with time. 
+Event consumers should not rely on the timing of an event with a given Reason reflecting a consistent underlying trigger, 
+or the continued existence of events with that Reason. 
```

Comment:
Are you capitalizing the word `Reason` intentionally?

Reply:
Nope, lemme correct it